src/jormungandr/testing/jormungandr-integration-tests/src/networking/explorer.rs

```diff
@@ -61,40 +61,172 @@ pub fn passive_node_explorer() {
     let leader_1 = controller
         .spawn(SpawnParams::new(LEADER_1).in_memory())
         .unwrap();
-    let _leader_2 = controller
+    let leader_2 = controller
         .spawn(SpawnParams::new(LEADER_2).in_memory())
         .unwrap();
-    let _leader_3 = controller
+    let leader_3 = controller
         .spawn(SpawnParams::new(LEADER_3).in_memory())
         .unwrap();
 
     let passive = controller
         .spawn(SpawnParams::new(PASSIVE).passive().in_memory())
         .unwrap();
+
     let mut alice = controller.controlled_wallet(ALICE).unwrap();
-    let bob = controller.controlled_wallet(BOB).unwrap();
+    let mut bob = controller.controlled_wallet(BOB).unwrap();
+    let mut clarice = controller.controlled_wallet(CLARICE).unwrap();
+    let mut mem_pool_checks = Vec::new();
 
-    let mem_pool_check = FragmentSender::from(&controller.settings().block0)
-        .send_transaction(&mut alice, &bob, &leader_1, 1_000.into())
-        .unwrap();
+    mem_pool_checks.push(
+        FragmentSender::from(&controller.settings().block0)
```

Comment:
I think we duplicating this line. Why not create variable of FragmentSender type?